python/salpytools/salpylib.py

```diff
@@ -33,19 +32,22 @@
 # fuction load_SALPYlib()
 
 
-#SAL__CMD_COMPLETE=303
+# SAL__CMD_COMPLETE=303
 spinner = itertools.cycle(['-', '/', '|', '\\'])
 
-def create_logger(level=logging.NOTSET,name='default'):
+
+def create_logger(level=logging.NOTSET, name='default'):
     ''' Simple Logger '''
     logging.basicConfig(level=level,
                         format='[%(asctime)s] [%(levelname)s] %(message)s',
                         datefmt='%Y-%m-%d %H:%M:%S')
     logger = logging.getLogger(name)
     return logger
 
+
 # Create a logger for all functions
-LOGGER = create_logger(level=logging.NOTSET,name='SALPYLIB')
+LOGGER = create_logger(level=logging.NOTSET, name='SALPYLIB')
+
 
 def load_SALPYlib(Device):
     '''Trick to import modules dynamically as needed/depending on the Device we want'''
```

Comment:
Docstrings should use triple double quotes: https://developer.lsst.io/python/numpydoc.html#docstrings-must-be-delimited-by-triple-double-quotes

Eventually, they should also comply to the extent possible with numpydoc standards in order to maximize usability for customers of this library: https://developer.lsst.io/python/numpydoc.html

Reply:
Replaced all triple ''' by triple """